Library/Homebrew/utils/splat.rb

```diff
@@ -0,0 +1,20 @@
+# typed: false
+# frozen_string_literal: true
+
+module Utils
+  # Wrappers for Ruby core methods that accept splat arguments. This file is `typed: false` by design, but allows
```

Comment:
> This file is `typed: false` by design

Why is that? Is it impossible to type?

Reply:
Correct, a file with a method that accepts splat arguments cannot be typed (Error [7019](https://sorbet.org/docs/error-reference#7019)):
```
utils/splat.rb:17: Splats are only supported where the size of the array is known statically https://srb.help/7019
    17 |      Process.kill(signal, *pids)
              ^
```
I can imagine a very verbose typed implementation (perhaps a case statement on the length of `pids`, and raising an error above some threshold?), but that isn't worth the trouble imho.

Reply:
Good to know, thanks!